{"name": "# C5162 Signal Strength section", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \n\n", "trace": "  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\C5162_Signal_Strength_section.py\", line 8, in sgnl_strngt_txt_hr\n    context.app.main_page.sgnl_strngt_txt_hr(sgnl_strngt_txt_hr)\n  File \"C:\\Everything\\IT\\Testing\\Automation_08_09_2019\\UI_automation_python_connectedIO\\pages\\main_page.py\", line 933, in sgnl_strngt_txt_hr\n    sgnl_strngt_txt_hr = wait.until(EC.visibility_of_element_located(SGNL_STRNGT_TXT_HR)).text\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 80, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "steps": [{"name": "Given Loginpage", "status": "passed", "start": 1624142795231, "stop": 1624142798011}, {"name": "Then Login with the given credentials", "status": "passed", "start": 1624142798011, "stop": 1624142800783}, {"name": "Then Click Dashboard menu icon and make sure Dashboard page reloaded", "status": "passed", "start": 1624142800783, "stop": 1624142800918}, {"name": "And Make sure Signal Strength is on the screen", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \n\n", "trace": "  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\C5162_Signal_Strength_section.py\", line 8, in sgnl_strngt_txt_hr\n    context.app.main_page.sgnl_strngt_txt_hr(sgnl_strngt_txt_hr)\n  File \"C:\\Everything\\IT\\Testing\\Automation_08_09_2019\\UI_automation_python_connectedIO\\pages\\main_page.py\", line 933, in sgnl_strngt_txt_hr\n    sgnl_strngt_txt_hr = wait.until(EC.visibility_of_element_located(SGNL_STRNGT_TXT_HR)).text\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 80, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "start": 1624142800918, "stop": 1624142819151}, {"name": "Then Mouse hover question mark in top right of the Signal Strength block and make sure tooltip appears", "status": "skipped", "start": 1624142821275, "stop": 1624142821275}, {"name": "Then The inscription No Data Available is in the center of Signal Strength block", "status": "skipped", "start": 1624142821275, "stop": 1624142821275}], "start": 1624142793879, "stop": 1624142821275, "uuid": "7e7be2af-2e4b-4f60-b46b-744caad93f94", "historyId": "5eaa07d92f5cb375ab7fe2d916367e55", "fullName": "# C5162 Signal Strength section: # C5162 Signal Strength section", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "# C5162 Signal Strength section"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}